Makefile

```diff
@@ -49,24 +39,15 @@ TEST_REQUIRES_ROOT_PACKAGES=$(filter \
 
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress
-ifneq ("$(GOOS)", "windows")
-	COMMANDS += containerd-shim
-endif
 BINARIES=$(addprefix bin/,$(COMMANDS))
-ifeq ("$(GOOS)", "windows")
-	BINARY_SUFFIX=".exe"
-endif
 
 GO_TAGS=$(if $(BUILDTAGS),-tags "$(BUILDTAGS)",)
 GO_LDFLAGS=-ldflags "-X $(PKG)/version.Version=$(VERSION) -X $(PKG)/version.Revision=$(REVISION) -X $(PKG)/version.Package=$(PKG) $(EXTRA_LDFLAGS)"
 
-# go test -race is only supported on the patforms listed below.
 TESTFLAGS_RACE=
-ifeq ($(filter \
-    linux/amd64 freebsd/amd64 darwin/amd64 windows/amd64, \
-    $(GOOS)/$(GOARCH)),$(GOOS)/$(GOARCH))
-	TESTFLAGS_RACE= -race
-endif
+
+#include platform sepcific makefile
+include Makefile.${GOOS}
```

Comment:
Given we use parenthesis instead of curly brace as a variable delimiter throughout this Makefile, seems like we should use it for consistency here (even though functionally it doesn't matter)